Minor cleanup and additions to git remotes lecture material

ethanwhite opened the pull request.

brymz commented on materials/version-control-R.md:
```diff
@@ -234,19 +236,25 @@ add_size_classes <- function(df) {
     * Show on local, not remote
 * Push
     * Show on remote
+* Draw push/pull on diagram on board
```
This could be put in a `>` block as an action item.

brymz commented on materials/version-control-R.md:
```diff
@@ -234,19 +236,25 @@ add_size_classes <- function(df) {
 * Show history
 * Run code
 
+* Show an example of a working repository
```
`>` for these also.

ethanwhite commented:
Thanks @brymz. I fixed the two things you pointed out and also added a new set of instructor notes on how to handle the transition from Exercise 6 to Exercise 7.
